docs/development-workflow.md

```diff
@@ -51,6 +51,18 @@ or limit it as before with:
 DEBUG=calypso:* make run
 ```
 
+## Clearing browser data
+
+Seeing some strange errors that only happen on _your_ browser? You may need to clear your browser's local storage and database. Calypso saves a lot here and if you've changed the way your data is kept (especially in redux state), it can break things in unepxected ways during the transition.
+
+At the browser console line:
+
+```js
+localStorage.clear(); indexedDB.deleteDatabase( 'calypso' );
+```
+
+And this should clear out your database. Just make sure that in your changes you're not creating problems for production users!
```

Comment:
Clearing local storage doesn't cause issues for other users, but it could fix issues for you in a way that leaves it broken for others. I guess that's what I was getting at.